packages/material-ui/scripts/rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
```

Comment:
We are already using https://github.com/ai/size-limit. I believe it's answering the same problem. Shouldn't we use a single tool?

Reply:
size limit creates limits. My tool is about showing different sizes.
1. size of bundle processed with bundler
1. size of bundle loader with browser
1. size of bundle parsed with browser
1. treeshaked size for esm bundles

For it's more important to see these values than moving limits over and over again.

Reply:
Does it have a mode where it only outputs and doesn't write in the filesystem? I'm wondering about constantly having to commit the snapshot.

Reply:
Isn't it the point to show users these numbers?

Reply:
I figured out that I can live with that 👍